modules/api/src/main/java/net/countercraft/movecraft/MovecraftRepair.java

```diff
@@ -155,11 +157,13 @@ public boolean saveRegionRepairState(World world, ProtectedRegion region) {
 
     public Clipboard loadCraftRepairStateClipboard(Craft craft, Sign sign) {
         File dataDirectory = new File(plugin.getDataFolder(), "CraftRepairStates");
-        String repairName = craft.getNotificationPlayer().getName();
-        repairName += "_";
-        repairName += ChatColor.stripColor(sign.getLine(1));
+        File playerDirectory = new File(dataDirectory,craft.getNotificationPlayer().getUniqueId().toString());
```

Comment:
Does this create the folder if it does not already exist?  If not then that should be fixed.

Reply:
It by itself does not create a new folder by itself. A new directory is created by calling the mkdirs() method in a File instance.